# Fix prefix and helpdir paths
# Fix compiler options <jscott@posteo.net>
--- mmail-0.52/Makefile.orig	2020-01-07 18:42:09.532570753 -0300
+++ mmail-0.52/Makefile	2020-01-07 18:54:50.047222097 -0300
@@ -9,16 +9,16 @@ isrc = interfac
 # is for any post-processing that needs doing.
 
 ifeq ($(DEBUG),Y)
-	OPTS = -g -Wall -pedantic
+	CXXFLAGS += -g -Wall -pedantic
 else
-	OPTS = -O2 -Wall -pedantic
+	CXXFLAGS += -O2 -Wall -pedantic
 	POST = strip mm$(E)
 endif
 
 # PREFIX is the base directory under which to install the binary and man
 # page; generally either /usr/local or /usr (or perhaps /opt...).
 
-PREFIX = /usr/local
+PREFIX = $(DESTDIR)/usr
 
 #--------------------------------------------------------------
 # Defaults are for the standard curses setup:
@@ -72,7 +72,7 @@ ifeq ($(SYS),DOS)
 		c:\djgpp\bin\cwsdstub.exe+mm mm.exe; del mm
 endif
 
-HELPDIR = $(PREFIX)/man/man1
+HELPDIR = $(PREFIX)/share/man1
 O = o
 
 .SUFFIXES: .cc
@@ -88,10 +88,10 @@ arealist.o letterl.o letterw.o lettpost.
 tagline.o help.o main.o
 
 $(MOBJS) : %.o: $(msrc)/%.cc
-	$(CXX) $(OPTS) -c $<
+	$(CXX) $(CXXFLAGS) -c $<
 
 $(IOBJS) : %.o: $(isrc)/%.cc
-	$(CXX) $(OPTS) -I$(CURS_DIR) -c $<
+	$(CXX) $(CXXFLAGS) -I$(CURS_DIR) -c $<
 
 mm$(E):	$(MOBJS) $(IOBJS)
 	$(CXX) -o mm$(E) $(MOBJS) $(IOBJS) $(LIBS)
@@ -106,7 +106,8 @@ clean:
 	$(RM) mm$(E)
 
 install::
-	install -c -s mm $(PREFIX)/bin
+	install -d $(PREFIX)/bin $(HELPDIR)
+	install -c $(INSTALL_OPTS) mm $(PREFIX)/bin
 	install -c -m 644 mm.1 $(HELPDIR)
 	$(RM) $(HELPDIR)/mmail.1
 	ln $(HELPDIR)/mm.1 $(HELPDIR)/mmail.1
